src/main/java/com/pinterest/secor/uploader/Uploader.java

```diff
@@ -113,7 +122,9 @@ protected void uploadFiles(TopicPartition topicPartition) throws Exception {
                 mFileRegistry.deleteTopicPartition(topicPartition);
                 mZookeeperConnector.setCommittedOffsetCount(topicPartition, lastSeenOffset + 1);
                 mOffsetTracker.setCommittedOffsetCount(topicPartition, lastSeenOffset + 1);
-
+                if (mConfig.getOffsetsStorage().equals("kafka")) {
```

Comment:
Can this constant "kafka" defined in a a central place, e.g. SecorConfig.java?  And should it be 'equals' or 'equalsIgnoreCase'?

Reply:
@HenryCaiHaiying yeah I was actually thinking the same thing. I think Constants way is better.

Reply:
Also make this check in uploader.init method, this check should only be done once since the config doesn't change.